File: discover_ruby/manipulate_the_data.rb
#!/usr/bin/ruby

require 'httpclient'
require 'json'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token e107aed5a9acf8df3246ffc645de998ef2046d5a'
}

http = HTTPClient.new
response = http.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')

ruby_projects = JSON.parse(response)
puts ruby_projects["items"].map {|index| index["full_name"]}

File: discover_more_python/manipulate_the_data.py
import urllib2
import json

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token bc38982215eef8560352730fdfc6ee82bd22beee'
}

response = urllib2.urlopen('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc')
html = response.read()

parsed_json = json.loads(html)

for i in range(len(parsed_json["items"])):
    print(parsed_json["items"][i]["full_name"])



